### PR TITLE
Updating Logic App Templates

### PR DESCRIPTION
**Pull request by jeffhollan**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
* Updated all Logic App templates to use the current schema 2015-08-01-preview

### Description of the change
All the Logic App templates now use the current connectors and schema

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jeffhollan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jehollan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-logic-app-ftp-to-blob/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        "Free",
+        "Basic",
+        "Standard",
+        "Premium"
+      ],
+      "metadata": {
+        "description": "The pricing tier for the logic app."
+      }
+    },
+    "azureblobaccountName": {
```
Comment (singhkay) [inline 101-logic-app-ftp-to-blob/azuredeploy.json]: camelCase names

File: 101-logic-app-ftp-to-blob/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      "type": "string",
+      "metadata": {
+        "description": "Account name of the Azure Blob storage account."
+      }
+    },
+    "azureblobaccessKey": {
```
Comment (singhkay) [inline 101-logic-app-ftp-to-blob/azuredeploy.json]: camelCase

File: 101-logic-app-sendgrid/azuredeploy.json [hunk 3]
```diff
@@ -60,30 +55,28 @@
...
-        "product": "sendgrid_azure",
-        "promotionCode": ""
-      },
+      "name": "[parameters('sendgridName')]",
       "properties": {
-        "password": "[parameters('sendgridPassword')]",
-        "acceptMarketingEmails": "0",
-        "email": ""
+        "api": {
+          "id": "[concat('/subscriptions/80d4fe69-c95b-4dd2-a938-9250f1c8ab03/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/sendgrid')]"
```
Comment (singhkay) [inline 101-logic-app-sendgrid/azuredeploy.json]: Does the sub id need to be hardcoded?

**Comment by singhkay** [comment]
@jeffhollan Few minor comments

**Comment by singhkay** [comment]
@jeffhollan any update on this?

**Comment by jeffhollan** [comment]
@singhkay yes sorry got held up on other things - you are right on feedback. Will make changes next week and append branch and we can merge

**Comment by singhkay** [comment]
@jeffhollan Looks like there is an error deploying https://travis-ci.org/Azure/azure-quickstart-templates/builds/146788991#L239

**Comment by singhkay** [comment]
merged

**Pull request merged**